Assets/Scripts/Internal/PlayerGPS.cs

```diff
@@ -385,27 +385,10 @@ public NameHelper.BankTypes GetNameBankOfCurrentRegion()
 
         /// <summary>
         /// Gets the dominant race in player's current region.
-        /// This seems to be based on subclimate rather than FACTION.TXT.
-        /// The faction data has very little diversity and does not match observed race in many regions.
         /// </summary>
         public Races GetRaceOfCurrentRegion()
         {
-            // Racial distribution in Daggerfall:
-            //  * Desert, Desert2, Rainforest = Redguard
-            //  * Mountain, MountainWoods = Nord
-            //  * Swamp, Subtropical, Woodlands, Default = Breton
-
-            DFLocation.ClimateSettings settings = MapsFile.GetWorldClimateSettings(climateSettings.WorldClimate);
-            switch(settings.People)
-            {
-                case FactionFile.FactionRaces.Redguard:
-                    return Races.Redguard;
-                case FactionFile.FactionRaces.Nord:
-                    return Races.Nord;
-                case FactionFile.FactionRaces.Breton:
-                default:
-                    return Races.Breton;
-            }
+            return (Races) MapsFile.RegionRaces[GameManager.Instance.PlayerGPS.CurrentRegionIndex] + 1;
```

Comment:
I don't agree with this change. Classic definitely generates Nord NPCs around those subclimates, and RegionRaces array only defines Breton or Redguard (0 or 1). This change would remove that diversity and will not match classic. For example, NPC selection in Wrothgarion Mountains/Whitetry in classic and DFU with existing code will match:

**Classic**
![fall_102](https://user-images.githubusercontent.com/10426244/87876001-442c4800-ca18-11ea-8566-39fd8138411f.png)


**DFU**
![image](https://user-images.githubusercontent.com/10426244/87875997-3f679400-ca18-11ea-8957-17ac700f069d.png)


And then after this code change the NPCs no longer match.

**After Edit**
![image](https://user-images.githubusercontent.com/10426244/87876056-a84f0c00-ca18-11ea-95d6-d5284c8b405b.png)


Happy to take the %lp macro, just not with this edit included. Perhaps we need to rethink the accuracy of MapsFile.RegionRaces instead.